database/aoai-proxy.sql

```diff
@@ -288,8 +284,6 @@ CREATE TABLE aoai.event (
     organizer_email character varying(128) NOT NULL,
     event_url character varying(256) NOT NULL,
     event_url_text character varying(256) NOT NULL,
-    max_token_cap integer NOT NULL,
```

Comment:
@aaronpowell max_token_cap needs to stay as it's used to override a user specified max_token